Makefile

```diff
@@ -38,12 +38,16 @@ CGOFLAG ?= CGO_ENABLED=1
 # should be an absolute directory as it is used by e/Makefile too, from the e/ directory.
 RELEASE_DIR := $(CURDIR)/$(BUILDDIR)/artifacts
 
+LDFLAGS ?= -w -s $(KUBECTL_SETVERSION)
+
+# Appending new conditional settings for community build type
+LDFLAGS += $(if $(filter $(TELEPORT_BUILD_TYPE),community),-X github.com/gravitational/teleport/lib/modules.teleportBuildType=community)
```

Comment:
How/under what conditions does this get set to `ent`?

Reply:
When running an enterprise build, it will always [return as `ent`](https://github.com/gravitational/teleport.e/blob/master/tool/modules/modules.go#L168), so we don't need to set it anywhere